scripts/tasks/3_annotate_bad_channels.ipynb

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
<p>Bad channels are already annotated in the previous step, alongside the bad segments; and are stored in the info.</p><p>We can split step 2 into 2, bad segments and bad channels, but it's a bit excessive.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mscheltienne/eeg-flow/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>